File: LeetCode/80. Remove Duplicates from Sorted Array II.py
###  KNOWLEDGE IS POWER  ###

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        for e in nums:
            if i < 2 or e > nums[i-2]:
                nums[i] = e
                i += 1 
        return i 

File: LeetCode/80. Remove Duplicates from Sorted Array II.java
/***  KNOWLEDGE IS POWER  ***/

class Solution {
    public int removeDuplicates(int[] nums) {
        int i = 0 ;
        for (int e : nums)
            if (i < 2 || e > nums[i-2])
                nums[i++] = e ;
        return i;
    }
}
